pkg/server/loss_of_quorum.go

```diff
@@ -21,8 +21,43 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/util/stop"
 	"github.com/cockroachdb/errors"
+	"github.com/cockroachdb/pebble/vfs"
 )
 
+// NewRecoveryServer creates recovery server and underlying plan storage.
```

Comment:
This is just helper function inside server package that creates storage and server, maybe the naming could be improved or it could be split so that we have create storage and the pass it to loqrecovery.NewServer() if that's cleaner.